arraysum.c

```diff
@@ -0,0 +1,43 @@
+/*Sean Kee*/
+/*Sums all the numbers in the array except the one at the given index; Returns new array*/
```

Comment:
I think you got the instructions mixed up, but I can see how you interpreted it from the directions. Nicely done, given the way you did it!